homework4.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<head>
+</head>
+
+
+<body>
+<script>
+
+
+
+</script>
+</body>
+
```

Comment:
👍 Nice scaffolding! I would remove the empty space on line 13 to 19. Typically there should be no code between the `</body>` and `</html>` tag.